README.rst

```diff
@@ -679,7 +679,9 @@ the run step:
         # value is present it should be a map of additional properties that
         # should be added to the build artifacts.json file. The artifacts.json
         # file can be used to publish artifacts to another system (such as
-        # Gauntlet) with the accompanying metadata.
+        # Gauntlet) with the accompanying metadata. By default artifacts will be
+        # listed in the artifacts.json file this can be disabled by adding the
```

Comment:
`listed in the artifacts.json file this can be disabled by adding...` should maybe be something like `listed in the artifacts.json file; this can be disabled by adding...` or `listed in the artifacts.json file. This can be disabled by adding...`